packages/core/src/utils/WithStyledDraggables.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import 'twin.macro'
+import {DragIndicator} from '@styled-icons/material/DragIndicator'
+import Tippy, {TippyProps} from '@tippyjs/react'
+import {withDraggables} from '@udecode/plate-dnd'
+import {ELEMENT_PARAGRAPH} from '@udecode/plate-paragraph'
+import {ELEMENT_BLOCKQUOTE} from '@udecode/plate-block-quote'
+import {ELEMENT_H1, ELEMENT_H2, ELEMENT_H3} from '@udecode/plate-heading'
+import {ELEMENT_OL, ELEMENT_TODO_LI, ELEMENT_UL} from '@udecode/plate-list'
+import {ELEMENT_CODE_BLOCK} from '@udecode/plate-code-block'
+import {ELEMENT_MEDIA_EMBED} from '@udecode/plate-media-embed'
+import {ELEMENT_TABLE} from '@dreifuss-wysiwyg-editor/table'
+
+import 'tippy.js/dist/tippy.css'
+
+const GrabberTooltipContent = () => (
+  <div style={{fontSize: 12}}>
```

Comment:
Is it possible to center this handle in the middle of the elements it's suppose to drag. If you add a table the drag handle is on the top rather then the center.

Reply:
@roosnic1 yes I understand your point, but I think having the dragging pointer in the beginning of the node which can be moved is better .. like here
<img width="1640" alt="Screen Shot 2021-10-04 at 3 23 12 PM" src="https://user-images.githubusercontent.com/69328362/135860173-cc5c178a-2db1-4fed-a5b3-f6f09a3bff2a.png">

It could be confusing if every line from these arabic lines can be moved, but getting the pointer in the beginning makes it more clear. What do you think?